assets/js/base/hooks/payment-methods/use-active-payment-method.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { useCheckoutContext } from '@woocommerce/base-context/checkout-context';
+import { useEffect } from '@wordpress/element';
+import { getPaymentMethods } from '@woocommerce/blocks-registry';
+
+const useActivePaymentMethod = () => {
```

Comment:
Yea I can make sure the payment method component just receives the `isActive` prop, essentially something like:

```js
<PaymentMethod isActive={ activePaymentMethod === paymentMethodId } />
```

However, I think the hook might still be needed for any checkout ui/ux chrome (tabs?) that indicate what payment method is currently active.

Reply:
Looks like I'm already just passing along an `isActive` prop for the rendered payment method component: https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1349/commits/0b430a2e3370a15bb2410702e2716f2066627c86#diff-cea364bc272eb722c3373b4d6e828d14R69-R73

Reply:
> Looks like I'm already just passing along an isActive prop for the rendered payment method component

Yeah, that's why I pointed it out, since the `isActive` prop already exists, the hook is redundant. As you said, the "chrome" tabs component will need it, but it probably can be kept as a "provate" API.

Reply:
Oh, ya it definitely will be a private internal api.  Also not planning on exposing any of these hooks to the payment methods themselves, anything they need will be exposed via props, not hooks.